app/src/components/validators.js

```diff
@@ -278,9 +285,16 @@ const validators = {
       return true;
     },
 
+    delayTS: value => {
+      if (value) {
+        return validatorUtils.isInt(value);
+      }
```

Comment:
Would it be worth putting in an extra sanity check that the integer must be greater than 0?

Reply:
Good idea.  I'll get that in next time.  May even be a good idea to validate that it is a valid timestamp...?